pkg/store/verifiable/store.go

```diff
@@ -52,11 +65,15 @@ func (s *Store) SaveVC(vc *verifiable.Credential) error {
 		return fmt.Errorf("failed to put vc: %w", err)
 	}
 
+	if err := s.store.Put(name, []byte(vc.ID)); err != nil {
+		return fmt.Errorf("store vc name to id map : %w", err)
+	}
+
 	return nil
 }
 
-// GetVC get verifiable credential
-func (s *Store) GetVC(vcID string) (*verifiable.Credential, error) {
+// GetCredential retrieves a verifiable credential based on ID.
+func (s *Store) GetCredential(vcID string) (*verifiable.Credential, error) {
```

Comment:
vcID to ID? just matching other renames you made in this file.

Reply:
agree. will update it.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/c6d66bb04d814caeb92a753132b79a8818daa699..cc1f4e1f94378cce223bd1aa5655a9922e50e3fd#diff-94f07b1b8913fb6c9dbe128104e469c9R76).